scripts/bootstrap.sh

```diff
@@ -164,8 +228,8 @@ update_stamp_vmss() {
   exit_on_error "Execution of edX sandbox playbook failed"
   
   # oxa playbooks
-  $ANSIBLE_PLAYBOOK -i localhost, -c local -e@$OXA_PLAYBOOK_CONFIG $OXA_PLAYBOOK_ARGS $OXA_PLAYBOOK --tags "edxapp"
-  exit_on_error "Execution of OXA edxapp playbook failed"
+  #$ANSIBLE_PLAYBOOK -i localhost, -c local -e@$OXA_PLAYBOOK_CONFIG $OXA_PLAYBOOK_ARGS $OXA_PLAYBOOK --tags "edxapp"
+  #exit_on_error "Execution of OXA edxapp playbook failed"
```

Comment:
Was this disabled due to the patch failure? Patch is now disabled in /playbooks/roles/edxapp/tasks/main.yml, so you can re-enable this.